itou/job_applications/management/commands/reject_job_applications_after_delay.py

```diff
@@ -0,0 +1,42 @@
+from django.db import transaction
+from django.template import loader
+from django.template.defaultfilters import pluralize
+
+from itou.job_applications.enums import RefusalReason
+from itou.job_applications.models import JobApplication
+from itou.utils.command import BaseCommand
+
+
+class Command(BaseCommand):
+    def get_answer_text_from_file(self):
+        return loader.render_to_string("apply/refusal_messages/auto.txt")
+
+    def handle(self, **options):
+        answer = self.get_answer_text_from_file()
+
+        job_seekers_with_their_job_applications = (
+            JobApplication.objects.job_seekers_with_their_rejectable_applications()
+        )
+        job_application_ids = [
+            str(uuid) for item in job_seekers_with_their_job_applications for uuid in item["applications"]
+        ]
+        job_applications = JobApplication.objects.filter(id__in=job_application_ids)
+
+        for job_seeker in job_seekers_with_their_job_applications:
+            job_seeker_applications = job_applications.filter(job_seeker_id=job_seeker["job_seeker"])
+
+            with transaction.atomic():
+                job_seeker_applications.select_for_update()
+                for job_seeker_job_application in job_seeker_applications:
+                    job_seeker_job_application.refusal_reason = RefusalReason.AUTO
+                    job_seeker_job_application.refusal_reason_shared_with_job_seeker = True
+                    job_seeker_job_application.answer = answer
+                    job_seeker_job_application.refuse(user=None)
+
+        self.logger.info(
+            "%s auto rejected job applicationw%s for %s job seeker%s.",
+            job_applications.count(),
+            pluralize(job_applications.count()),
```

Comment:
Pour des logs, je ne pense pas que ça vaille le coût.
D'autant que quelqu'un cherchant naïvement la chaîne "auto rejected job applications for" dans nos logs ne trouvera les logs au singulier...